packages/circuit-ui/components/NotificationFullscreen/NotificationFullscreen.stories.tsx

```diff
@@ -0,0 +1,49 @@
+  // an image url should also work, for example: https://source.unsplash.com/QU-zhliIV8Q/1600x900
+  actions: {
+    primary: {
+      text: 'Look again',
+      onClick: () => console.log("It's still empty."),
+    },
+    secondary: {
+      text: 'Go elsewhere',
+      href: 'https://sumup.com',
+    },
+  },
```

Comment:
I'm really not sure about this. I wasn't sure about exposing the `ButtonGroup` directly but it might be easier than this complex prop in the end. What do you think?

Reply:
I love it 😃 I believe it's so much better than the ButtonGroup because it gives us more control over styling and aligning these buttons. I also don't think it's overly complex 👀